pkgs/top-level/all-packages.nix

```diff
@@ -6081,6 +6081,11 @@ in
 
   hyenae = callPackage ../tools/networking/hyenae { };
 
+  hylafaxp = callPackage ../servers/hylafaxp {
+    pam = null;
+    openldap = null;
```

Comment:
well, if those dependencies are never used in `hylafax`, could we not drop them?